.github/workflows/run_benchmarks.yml

```diff
@@ -0,0 +1,61 @@
+name: Run Benchmarks
```

Comment:
Eventually it would probably be useful to factor out this action into a repo for easier sharing: https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization

thats for later though